custom/ilsgateway/tanzania/handlers/soh.py

```diff
@@ -52,6 +52,7 @@ def _cleanup(s):
 class SohFormatter(Formatter):
 
     def format(self, text):
+        text = text.replace('\r', ' ').replace('\n', ' ').strip()
```

Comment:
ok, we can probably get rid of this line now that we do this in custom/ilsgateway/tanzania/handler.py